src/components/appHeading/AppHeading.tsx

```diff
@@ -54,7 +57,7 @@ export const AppHeading: React.FC<AppHeadingProps> = ({
         fontWeight={fontWeight}
         color={color}
         textAlign={textAlign}>
-        {text}
+        {limitText ? text.slice(0, limitText) + "..." : text}
```

Comment:
this code is wrong because it will add the three dots even for shorter texts.

also why are we using different code for the deploy page description field compared to the view jetton page description field?